src/PresentationalComponents/ClusterDetailHead/ClusterDetailHead.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, GridItem } from '@patternfly/react-core';
+import Skeleton from '@redhat-cloud-services/frontend-components/Skeleton';
+
+const ClusterDetailHead = ({ cluster, clusterFetchStatus }) => (
+  <React.Fragment>
+    <Grid md={6}>
+      <GridItem>
+        <div>
+          <span>UUID</span>
+          <span>
+            {clusterFetchStatus !== 'fulfilled' ? (
+              <Skeleton size="md" />
+            ) : (
+              cluster.cluster_id && 'Unknown'
```

Comment:
Can  be a translatable string here

Reply:
ooo and everywhere else, last seen etc, not gonna comment on each of em, just the one